sw/airborne/modules/nav/nav_smooth.c

```diff
@@ -194,8 +194,8 @@ bool snav_on_time(float nominal_radius)
   /* Recompute ground speeds every 10 s */
   if (ground_speed_timer == 0) {
     ground_speed_timer = 40; /* every 10s, called at 40Hz */
-    compute_ground_speed(airspeed, stateGetHorizontalWindspeed_f()->y,
-                         stateGetHorizontalWindspeed_f()->x); // Wind in NED frame
+    compute_ground_speed(airspeed, stateGetHorizontalWindspeed_f()->x,
```

Comment:
sorry, didn't look at the fix...
that part here was correct, only the naming in compute_ground_speed was misleading...